src/main/java/cz/metacentrum/perun/googlegroupconnector/GoogleGroupsServiceImpl.java

```diff
@@ -800,4 +826,286 @@ private void deleteMember(String groupName, String memberId) throws GoogleGroups
 		}
 	}
 
+
+	/**
+	 * New class representing tuple of team drive and its users from CSV file.
+	 */
+	public class DriveWithMembers {
```

Comment:
You don't need to create class, because this is a basically ``Map<TeamDrive,List<User>>``. Since this doesn't contains any additional methods or specific ``equals()`` etc., please use the Map. You can get keys, values or their pairs the same way like this list of ``DriveWithMembers``.